invesalius/gui/widgets/gradient.py

```diff
@@ -381,6 +381,279 @@ def _generate_event(self, event):
         )
         self.GetEventHandler().ProcessEvent(evt)
 
+class GradientNoSlide(wx.Panel):
+    # This widget is formed by a gradient background (black-white)
+    # Unlike GradientSlide, here the widget is used as a colorbar to display 
+    # the available colors (used in fmri support)
+    def __init__(self, parent, id, minRange, maxRange, minValue, maxValue, colour, colortype=0):
+        # minRange: the minimal value
+        # maxrange: the maximum value
+        # minValue: the least value in the range
+        # maxValue: the most value in the range
+        # colour: colour used in this widget.
+        super(GradientNoSlide, self).__init__(parent, id)
+        self._bind_events_wx()
+
+        self.min_range = minRange
+        self.max_range = maxRange
+        self.minimun = minValue
+        self.maximun = maxValue
+        self.selected = 0
+        self.colortype = colortype
+
+        self.min_position = 0
+        w, h = self.GetSize()        
+
+        self._gradient_colours = None
+        self.colour = colour
+
+    def _bind_events_wx(self):
+        self.Bind(wx.EVT_LEFT_DOWN, self.OnClick)
+        self.Bind(wx.EVT_LEFT_UP, self.OnRelease)
+        self.Bind(wx.EVT_PAINT, self.OutPaint)
+        self.Bind(wx.EVT_ERASE_BACKGROUND, self.OnEraseBackGround)
+
+        if sys.platform == "win32":
+            self.Bind(wx.EVT_LEAVE_WINDOW, self.OnLeaveWindow)
```

Comment:
This function self.OnLeaveWindow does not exist and because of that, InVesalius doesn't even start. Would you please check that this is correct.